paste/urlparser.py

```diff
@@ -1,31 +1,35 @@
 # (c) 2005 Ian Bicking and contributors; written for Paste (http://pythonpaste.org)
 # Licensed under the MIT license: http://www.opensource.org/licenses/mit-license.php
 """
-WSGI applications that parse the URL and dispatch to on-disk resources
+WSGI applications that parses the URL and dispatches to on-disk resources
```

Comment:
I think the previous statement was more grammatically correct: there are multiple applications in this file that each parse the URL and dispatch